test/TestUtilities/WindowUtil.cpp

```diff
@@ -3,6 +3,17 @@
 
 #include <ostream>
 
+std::string runDisplayTests()
+{
+#ifdef SFML_RUN_DISPLAY_TESTS
+    return "";
+#else
+    // https://github.com/catchorg/Catch2/blob/devel/docs/test-cases-and-sections.md#special-tags
+    // The "[.]" tag tells Catch2 to not run a given TEST_CASE
+    return "[.]";
```

Comment:
Wouldn't it be possible to make this more speaking, like `[.displaytest]` or similar?

Reply:
Good idea. I didn't know that was an option until I double-checked the Catch docs.